azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/PackageMojo.java

```diff
@@ -348,10 +389,15 @@ protected void copyLocalSettingsJson() throws IOException {
         log.info(SAVING_LOCAL_SETTINGS_JSON);
         final File sourceLocalSettingsJsonFile = getLocalSettingsJsonFile();
         final File destLocalSettingsJsonFile = Paths.get(getDeploymentStagingDirectoryPath(), LOCAL_SETTINGS_JSON).toFile();
-        copyFilesWithDefaultContent(sourceLocalSettingsJsonFile, destLocalSettingsJsonFile, DEFAULT_LOCAL_SETTINGS_JSON);
+        String defaultContent = MessageFormat.format(DEFAULT_LOCAL_SETTINGS_JSON, getWorkerRuntime());
```

Comment:
Do we need to overwrite users configuration for native artifact? As default content will not work if users have a local.settings.json in their workspace

Reply:
Yes, you can test locally without the host.json file.
local.settings.json is used for local development and will not be published to Azure. [On the other hand, host.json is used for configuring the Functions host process when you run projects locally and in Azure, for log level for instance]

https://learn.microsoft.com/en-us/azure/azure-functions/functions-develop-local

Reply:
@viniciusfcf @SandraAhlgrimm Thank you all for your help. Just want to confirm whether customHandler configuration is required in host.json? I just tried to remove following configuration and func seems broken. 
If its required, do we need to add validation and add following configuration into host.json? If so, we may need to add some extra logic to add following part to user's `host.json`, as currently we will generate it only if user did not have `host.json` in their workspace
```
    "customHandler": {
        "description": {
            "defaultExecutablePath": "quarkus-azure-functions-http-1.0.0-SNAPSHOT",
            "workingDirectory": "",
            "arguments": [
                "-Dquarkus.http.port=${FUNCTIONS_CUSTOMHANDLER_PORT}"
            ]
        },
        "enableForwardingHttpRequest": true
    }
```